tokens/alias/light.json

```diff
@@ -1435,6 +1445,13 @@
           "type": "color"
         }
       }
+    },
+    "borderRadius": {
```

Comment:
@davyd-akamai We can't have this be the same name as the global token. Maybe we rename this `radius`? Or can we just put this under `border.borderRadius`?